sof_wrapper/api/fhir.py

```diff
@@ -231,17 +231,20 @@ def route_fhir(relative_path, session_id):
     # use session lookup across sessions if necessary
     upstream_fhir_base_url = iss
     upstream_fhir_url = '/'.join((upstream_fhir_base_url, relative_path))
+
+    passthrough_headers = {'Authorization', 'Accept'}
     upstream_headers = {}
-    if 'Authorization' in request.headers:
-        upstream_headers = {'Authorization': request.headers['Authorization']}
+    for header_name in passthrough_headers:
+        if header_name in request.headers:
+            upstream_headers[header_name] = request.headers[header_name]
 
     upstream_response = requests.get(
         url=upstream_fhir_url,
         headers=upstream_headers,
         params=request.args,
     )
     upstream_response.raise_for_status()
-    return upstream_response.json()
+    return (upstream_response.text, upstream_response.status_code, upstream_response.headers.items())
```

Comment:
I'd suggest using flask's `make_request()` for all the other details it manages...